index.html

```diff
@@ -129,7 +129,7 @@
         <div class="follow-us" align="center"> 
 <a class="fa fa-facebook social-icon" href="https://www.facebook.com/MozilliansTamilNadu" target="_href"></a> 
 <a class="fa fa-twitter social-icon" href="https://twitter.com/mozillaTN" target="_href"></a> 
-<a class="fa fa-google-plus social-icon" href="#" target="_href"></a> </div>
+<a class="fa fa-google-plus social-icon" href="https://plus.google.com/+MozillaindiaOrg" target="_href"></a> </div>
 </footer>

```

Comment:
@drunkskunk the link you have added is Mozilla India but this website is about Mozilla Tamil Nadu